### ISSUE TITLE
複数行の文章で、エラー行が不正確 (Incorrect error line in multiple line text)

### ISSUE THREAD

**Issue by tomykaira** [textlint-ja/textlint-rule-no-doubled-conjunctive-particle-ga#11]
次のように、複数行のテキストで2行目に「が」が二回以上使われる場合、lineとして2が返ることを期待していますが、1が返ってきます。

```
@@ -59,5 +59,15 @@ tester.run("no-doubled-conjunctive-particle-ga", rule, {
                 }
             ]
         },
+        {
+            text: "こんにちは。\n今日は早朝から出発したが、定刻には間に合わなかったが、無事会場に到着した。",
+            errors: [
+                {
+                    message: `文中に逆接の接続助詞 "が" が二回以上使われています。`,
+                    line: 2,
+                    column: 12
+                }
+            ]
+        },
     ]
 });
```

```
  1) no-doubled-conjunctive-particle-ga
       こんにちは。
今日は早朝から出発したが、定刻には間に合わなかったが、無事会場に到着した。:

      AssertionError [ERR_ASSERTION]: line should be 2
      + expected - actual

      -1
      +2

      at /tmp/textlint-rule-no-doubled-conjunctive-particle-ga/node_modules/textlint-tester/src/test-util.ts:125:24
      at Array.forEach (<anonymous>)
      at /tmp/textlint-rule-no-doubled-conjunctive-particle-ga/node_modules/textlint-tester/src/test-util.ts:91:16

```

**Comment by takahashim** [comment]
ご指摘ありがとうございます！　修正します。
